### ISSUE TITLE
ANCOMBC updates

### ISSUE THREAD

**Issue by antagomir** [microbiome/mia#316]
ANCOMBC now [supports TreeSE](https://github.com/FrederickHuangLin/ANCOMBC/issues/32).

See that issue for details. 

Action points:
- update mia and/or mia tutorial accordingly
- once done, open issue to update OMA (may rely on mia updates (?) so better to do after)

We can discuss in Bioc Slack who will pick this up.

**Comment by antagomir** [comment]
Ok I think @HenrikEckermann said he would pick it up.

**Comment by antagomir** [comment]
I tend to think that example in OMA is sufficient. The mia vignette should be limited to tools implemented in the mia package itself, OMA is more general. We could however cite the ANCOMBC also in mia tutorial. But no examples (that would also require adding new Suggested packages etc. which is problematic)

**Comment by HenrikEckermann** [comment]
Indeed, I did install the DEV version for our last workshop and in there it already supported tse. However, it is possible that the output changed now. Also, there is a new function `ancombc2` for repeated measures and some other new features. Might be worth mentioning in one line. I will use ANCOMBC within the next 2-3 weeks and then I will be able to tell if we need to edit it in OMA due to changes in output/functionality. In the meantime, the OMA tutorial is still a great starting point for ANCOMBC the way it is. Let me know if you disagree. I could also adapt it earlier if necessary.

**Comment by antagomir** [comment]
Sounds good for now.

**Comment by antagomir** [comment]
@HenrikEckermann - do you have new comments now regarding possible ANCOMBC OMA updates?

**Comment by HenrikEckermann** [comment]
@antagomir just uploaded new pull request in OMA. I commented there:


...ANCOMBC has become quite complex by now with lots of functionality. The simple example that we cover in the OMA book does not need this complexity. So, I kept the simple example but make the reader aware of possibilities/new developments and reference to the elaborate tutorials by the developer. I think that is the best solution for now.
...
